locales/he_IL/LC_MESSAGES/duckduckgo.po

```diff
@@ -77,15 +79,17 @@ msgstr ".בנו %s"
 
 msgctxt "SERP footer content"
 msgid "%s%s%s Billion Searches"
-msgstr ""
+msgstr "%s%s%s מיליארד חיפושים"
 
 #. The placeholder here will be a bold number. Sorry, this is hard to translate.
 msgctxt "homepage ATB social proof"
 msgid "%s%s%s reviews"
 msgstr ""
 
 msgctxt "install-duckduckgo"
-msgid "%sClick %sAllow%sClick %sAdd%sCheck %sAllow%s, then click %sOkay%s"
+msgid ""
+"%sClick %sContinue To Installation%sClick %sAdd%sCheck %sAllow%s, then click "
+"%sOkay%s"
```

Comment:
Please, do not change the original strings. Original strings are tagged `msgid`, while the translations are `msgstr`.

```suggestion
msgid "%sClick %sAllow%sClick %sAdd%sCheck %sAllow%s, then click %sOkay%s"
```